sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -35,14 +78,14 @@ service-interface-as-public: true
 custom-strongly-typed-header-deserialization: true
 generic-response-type: true
 custom-types-subpackage: models
-custom-types: DocumentFormulaKind,DocumentAnnotationKind,DocumentPageKind,DocumentBarCodeKind,DocumentPageKind,FontStyle,FontWeight
+custom-types: DocumentFormulaKind,DocumentAnnotationKind,DocumentPageKind,DocumentBarcodeKind,FontStyle,FontWeight,ParagraphRole,DocumentSignatureType,DocumentTableCellKind,AddressValue,CurrencyValue,DocumentLanguage,DocumentSpan,DocumentStyle
 required-fields-as-ctor-args: true
 enable-sync-stack: true
 polling: {}
+output-model-immutable: true
```

Comment:
This will generate the models with private constructor and no setters. Setting this option now for an already GA'd service will result in backward incompatible changes. So, we should remove this setting to maintain backward compatibility.

Reply:
This should only impact the impl models. And as for the other custom-types, they are all introduced in this beta.